packages/common/src/store/tipping/slice.ts

```diff
@@ -26,10 +24,8 @@ const initialState: TippingState = {
     error: null,
     source: 'profile'
   },
-  recentTips: [],
-  storage: null,
   tipToDisplay: null,
-  showTip: true
+  showTip: false
```

Comment:
Set this as false initially so that in the case where we're dismissed, we don't show a loading tile